docs/en_US/Release.md

```diff
@@ -1,5 +1,109 @@
 # ChangeLog
 
+## Release 1.2 - 12/02/2019
+
+### Major Features
+* Feature Engineering
+  - New feature engineering interface
+  - Feature selection algorithms: Gradient feature selector & GBDT selector
+* NAS
+  - New NAS interface
+  - NAS algorithms: ENAS, DARTS, P-DARTS (in PyTorch)
+  - NAS in classic mode (each trial runs independently)
+* Model compression
```

Comment:
support the API for exporting compressed model

Reply:
Thanks, added.